src/components/InputTooltip.js

```diff
@@ -27,6 +27,8 @@ class InputTooltip extends Component {
         const { onAccept } = this.props;
 
         if (event.key === ENTER_KEY) {
+            event.preventDefault();
```

Comment:
this might not be obviuous here, why we call it.
I think a comment could be helpful